lib/sinon/stub-non-function-property.js

```diff
@@ -3,15 +3,13 @@
 var valueToString = require("./util/core/value-to-string");
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
-function stubNonFunctionProperty(object, property, value) {
+function stubNonFunctionProperty(object, property) {
```

Comment:
> This should throw like the other stub method with too many arguments.

It is internal only, and is called inside `sandbox.stub`, which does throw for too many arguments.